ci-operator/config/openshift/kueue-operator/openshift-kueue-operator-release-1.0.yaml

```diff
@@ -44,6 +52,12 @@ images:
       - destination_dir: .
         source_path: /usr/bin/oc
   to: src-with-oc-and-kubectl
+releases:
```

Comment:
why is this necessary?

Reply:
I guess one of the `Make` commands was throwing an error for me. I added this piece to appease that warning.